homework.js

```diff
@@ -1,22 +1,46 @@
 // 1. Set the variable `givenName` to the string "Addison".
+let givenName = 'Addison';
+console.log('Addison');
 
 // 2. Set candies equal to 20, people to 6, and leftover equal
 // to the remainder of dividing 20 by 6.
+let candies = 20;
+let people = 6;
+let leftover = 20 % 6;
+console.log('leftover');
+
```

Comment:
When you put something in quotation marks, that makes it a string. So your console will show literally that string of characters: `leftover`. If you want to see the value in the variable `leftover`, then you want to type it without the quotation marks: `console.log(leftover)`